From 1939276efa4a12ef9e14ea139a525c887de2bb77 Mon Sep 17 00:00:00 2001
From: Joel Stanley <joel@jms.id.au>
Date: Thu, 25 Oct 2018 15:18:56 +1030
Subject: [PATCH] init: Only print shutdown messages to log

We disable them from appearing on the console so users are not confused
as the system kexec's to the next kernel. This suppresses these
messages:

 The system is going down NOW!
 Sent SIGTERM to all processes
 Sent SIGKILL to all processes

Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 init/init.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/init/init.c b/init/init.c
index efab5dcb44c0..afac0fd7dee5 100644
--- a/init/init.c
+++ b/init/init.c
@@ -762,16 +762,16 @@ static void run_shutdown_and_kill_processes(void)
 	 * shut things down gracefully... */
 	run_actions(SHUTDOWN);
 
-	message(L_CONSOLE | L_LOG, "The system is going down NOW!");
+	message(L_LOG, "The system is going down NOW!");
 
 	/* Send signals to every process _except_ pid 1 */
 	kill(-1, SIGTERM);
-	message(L_CONSOLE, "Sent SIG%s to all processes", "TERM");
+	message(L_LOG, "Sent SIG%s to all processes", "TERM");
 	sync();
 	sleep1();
 
 	kill(-1, SIGKILL);
-	message(L_CONSOLE, "Sent SIG%s to all processes", "KILL");
+	message(L_LOG, "Sent SIG%s to all processes", "KILL");
 	sync();
 	/*sleep1(); - callers take care about making a pause */
 }
-- 
2.32.0

